doc/pseudocode.md

```diff
@@ -0,0 +1,187 @@
+# SQL Query Builder Pseudocode 
+
+## Task 1: UI structure and SQL input field
+public class QueryBuilderUI
+{
+    public void Initialize()
+    {
+        SetupUILayout();
+        AddSQLInputField();
+    }
+    private void SetupUILayout()
+    {
+        // Create UI components: Tables, Joins, SQL Query, Columns, Filters, Sorting, Query Result
+    }
+    private void AddSQLInputField()
+    {
+        // Add a static SQL query input field where users can manually enter queries
+    }
+}
+
+## Task 2: Selecting a single table
+public class TableSelection
```

Comment:
make no sense to create lot of classes
for instance why each of them has private method UpdateSQL query?
How this should affect final SQL statement?